notebooks/official/experiments/comparing_local_trained_models.ipynb

```diff
@@ -398,9 +398,14 @@
         "3. In the **Service account name** field, enter a name, and\n",
```

Comment:
<p>Split into 4 separate sections</p><p>Move Dataset after Objective.</p><p>Add missing "uses" (see template) in Objective</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/854/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>